### PR TITLE
Sorted order source

### PR DESCRIPTION
**Pull request by CasperMok**
- Sort source order upon the stream-based settings
- Scrollable input screen
- Fix typo, change alphaNunmeric -> alphaNumeric

### CODE REVIEW

File: LocalPackages/DolbyIORTSUIKit/Sources/DolbyIORTSUIKit/Private/ViewModels/StreamViewModel.swift [hunk 1]
```diff
@@ -16,8 +16,10 @@ final class StreamViewModel: ObservableObject {
         static let interactivityTimeOut: CGFloat = 5
     }
 
+    private let settingsManager: SettingsManager
     private let streamCoordinator: StreamCoordinator
     private var subscriptions: [AnyCancellable] = []
+    private var settingsSubscriptions: [AnyCancellable] = []
```
Comment (aravind-raveendran) [inline LocalPackages/DolbyIORTSUIKit/Sources/DolbyIORTSUIKit/Private/ViewModels/StreamViewModel.swift:22]: we can use the same `subscriptions` for all subscriptions in the view model

File: LocalPackages/DolbyIORTSUIKit/Sources/DolbyIORTSUIKit/Private/ViewModels/StreamViewModel.swift [hunk 2]
```diff
@@ -74,6 +79,9 @@ final class StreamViewModel: ObservableObject {
                 guard let self = self else { return }
                 switch state {
                 case let .subscribed(sources: sources, numberOfStreamViewers: _, streamDetail: _):
+                    if self.sources.isEmpty, let source = sources.first {
```
Comment (aravind-raveendran) [inline LocalPackages/DolbyIORTSUIKit/Sources/DolbyIORTSUIKit/Private/ViewModels/StreamViewModel.swift:82]: Looking at the conditions - if the first condition is true`self.sources.isEmpty` there won't be a first source anyway ?

File: LocalPackages/DolbyIORTSUIKit/Sources/DolbyIORTSUIKit/Private/ViewModels/StreamViewModel.swift [hunk 3]
```diff
@@ -74,6 +79,9 @@ final class StreamViewModel: ObservableObject {
...
+                        settingsManager.setActiveSetting(for: .stream(streamID: source.streamId))
```
Comment (aravind-raveendran) [inline LocalPackages/DolbyIORTSUIKit/Sources/DolbyIORTSUIKit/Private/ViewModels/StreamViewModel.swift:83]: may be we use `streamDetail` for the setting now ? If so we don't have to check the first item in the array ?

File: LocalPackages/DolbyIORTSUIKit/Sources/DolbyIORTSUIKit/Private/ViewModels/StreamViewModel.swift (hunk 1 above)
Comment (CasperMok) [reply LocalPackages/DolbyIORTSUIKit/Sources/DolbyIORTSUIKit/Private/ViewModels/StreamViewModel.swift:22]: Got it

File: LocalPackages/DolbyIORTSUIKit/Sources/DolbyIORTSUIKit/Private/ViewModels/StreamViewModel.swift (hunk 2 above)
Comment (CasperMok) [reply LocalPackages/DolbyIORTSUIKit/Sources/DolbyIORTSUIKit/Private/ViewModels/StreamViewModel.swift:82]: Will fix it

File: LocalPackages/DolbyIORTSUIKit/Sources/DolbyIORTSUIKit/Private/ViewModels/StreamViewModel.swift (hunk 3 above)
Comment (CasperMok) [reply LocalPackages/DolbyIORTSUIKit/Sources/DolbyIORTSUIKit/Private/ViewModels/StreamViewModel.swift:83]: Will fix it

File: LocalPackages/DolbyIORTSUIKit/Sources/DolbyIORTSUIKit/Private/ViewModels/StreamViewModel.swift (hunk 2 above)
Comment (aravind-raveendran) [reply LocalPackages/DolbyIORTSUIKit/Sources/DolbyIORTSUIKit/Private/ViewModels/StreamViewModel.swift]: Cool right, I read it wrong !!

**Review by aravind-raveendran — APPROVED** [review]

**Pull request merged**